engine/src/main/java/org/hibernate/search/engine/backend/analysis/AnalyzerNames.java

```diff
@@ -19,14 +19,59 @@ private AnalyzerNames() {
 	 * <p>
 	 * This analyzer will generally be used for full-text field that don't require specific analysis.
 	 * <p>
-	 * Unless overridden by explicit analysis configuration, the default analyzer will be the standard analyzer:
+	 * Unless overridden by explicit analysis configuration,
+	 * the default analyzer will be the {@link #STANDARD} analyzer.
+	 */
+	public static final String DEFAULT = "default";
+
+	/**
+	 * The standard analyzer.
 	 * <ul>
 	 *     <li>First, tokenize using the standard tokenizer, which follows Word Break rules from the
 	 *     Unicode Text Segmentation algorithm, as specified in
 	 *     <a href="http://unicode.org/reports/tr29/">Unicode Standard Annex #29</a>.</li>
+	 *     <li>Then, it removes most punctuation.</li>
+	 *     <li>Finally, lowercase each token.</li>
+	 * </ul>
+	 */
+	public static final String STANDARD = "standard";
+
+	/**
+	 * The simple analyzer.
+	 * <ul>
+	 *     <li>First, tokenize the text into tokens whenever it encounters a character which is not a letter.</li>
 	 *     <li>Then, lowercase each token.</li>
 	 * </ul>
 	 */
-	public static final String DEFAULT = "default";
+	public static final String SIMPLE = "simple";
+
+	/**
+	 * The whitespace analyzer.
+	 * <ul>
+	 *     <li>Tokenize the text into tokens whenever it encounters a character which is not a white space.</li>
+	 *     <li>Do not change the tokens.</li>
+	 * </ul>
+	 */
+	public static final String WHITESPACE = "whitespace";
+
+	/**
+	 * The stop analyzer.
+	 * <ul>
+	 *     <li>First, tokenize the text into tokens whenever it encounters a character which is not a letter.</li>
+	 *     <li>Then, it removes english stop words.</li>
+	 *     <li>Finally, lowercase each token.</li>
+	 * </ul>
+	 */
+	public static final String STOP = "stop";
+
+	/**
+	 * The keyword analyzer.
+	 * <p>
+	 * Do not change in any way the text.
+	 * With this analyzer a full text field would behave exactly as it was a keyword field.
```

Comment:
`With this analyzer a full text field would behave similarly to a keyword field, but with fewer features: no terms aggregations, for example.`